LPD-42985 SF

sandrodw3 opened the pull request.

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

sandrodw3 commented:
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

sandrodw3 commented:
Pull request submitted to https://github.com/ealonso/liferay-portal/pull/2569. [See changes here.](https://github.com/sandrodw3/liferay-portal/compare/276f0077d72b~1...276f0077d72b) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

balazssk committed:
LPD-42985 Add fileEntry level subscription to DLApp

balazssk committed:
LPD-42985 ant build-service-portlet-documentlibrary

balazssk committed:
LPD-42985 Implement subscription wrapper

balazssk committed:
LPD-42985 Add MVCActioncommands for DLFileEntry

balazssk committed:
LPD-42985 semver

balazssk committed:
LPD-42985 Create dropdown option - subscribe/unsubscribe to DLFileEntry

balazssk committed:
LPD-42985 Create dropdown option - subscribe/unsubscribe to DLFolder

balazssk committed:
LPD-42985 SF rename

balazssk committed:
LPD-42985 Add resource permission

balazssk committed:
LPD-42985 Extend DLSubscriptionUtil with a method that determines if a FileEntry has an explicite or implicite subscription

balazssk committed:
LPD-42985 Add info panel subscription for File Entries

balazssk committed:
LPD-42985 Unsubscribable depends on ancestors, not the current folder

balazssk committed:
LPD-42985 Migrate info panel un/subscribe icon to use clay

balazssk committed:
LPD-44346 Seperate View File Entry Info Panel from Admin Info Panel

balazssk committed:
LPD-44346 New function to add DocumentFolders

balazssk committed:
LPD-44346 Folder checkbox is at a different part of the DOM

JhossephG committed:
LPD-44524 Adding original value from custom fields to system objects payload json

JhossephG committed:
LPD-44524 Return new field added in json when oldValue is true

balazssk committed:
LPD-44346 Tests

balazssk committed:
LPD-42985 Root folder is always unsubscribable

JhossephG committed:
LPD-44524 Add test

balazssk committed:
LPD-44346 SF - use File Entry and Folder in naming

balazssk committed:
LPD-44346 SF - rename

balazssk committed:
LPD-42985 Use monospaced attribute instead of css

devpedrotavares committed:
LPD-44779 Execute udpate actions when the status of the entry is updated

ctang3 committed:
LPD-44987 Add finder to create a unique index that includes ctCollectionId

ctang3 committed:
LPD-44987 Add constraint resolver for AssetEntryAssetCategoryRel

ctang3 committed:
LPD-44987 buildLang

ctang3 committed:
LPD-44987 buildService

devpedrotavares committed:
LPD-44779 Add test to assert that onAfterUpdate actions are triggered with workflow status changes

Commit:
LRSD-7806 Add 'Go to DEV' option to the Liferay SaaS Activation tab

pedro-oliveira446 committed:
LPD-44371 Create new public method to deploy all object definitions

pedro-oliveira446 committed:
LPD-44371 Integration Test

vngoliveira committed:
LPD-45168 Verify the value length before saving it in the column during the upgrade process

vngoliveira committed:
LPD-45168 Update upgrade test

vngoliveira committed:
LPD-45168 Exclude classes that use org.jsoup from SF rule LPD-42623

michelevigilante committed:
LPD-45186 Fix passed quantity

lfbesada committed:
LPD-1226 Use body when there is no-element with the ID "content"

lfbesada committed:
LPD-1226 Add a warning trace

lfbesada committed:
LPD-1226 SF, move to the suitable package

lfbesada committed:
LPD-1226 Add a JUnit test to assert the correct behavior when the theme has an element with ID "content"

lfbesada committed:
LPD-1226 Add a test to assert that the content of the body tag is fully replaced when  can't find any element with ID "content"

lfbesada committed:
LPD-1226 Add assert to verify the warning trace

lfbesada committed:
LPD-1226 Apply the same fix when previewing the layout content to prevent a NPE in case no element with ID "content"

lfbesada committed:
LPD-45260 Apply the layout theme instead of the layout set theme

lfbesada committed:
LPD-45260 Set the theme display look and feel depending on the layout theme

lfbesada committed:
LPD-45260 Add an integration test for this use case

4lejandrito committed:
LPD-45266 Making test more robust to make it fail

The fragment is not being rendered in the page.

4lejandrito committed:
LPD-45266 Avoiding pollution of permission thread local

During the fragment rendering there are some scenarios in which the
request being passed to RESTClient is not authenticated. When that
happens the PermissionThreadLocal is set with the guest user inside the
REST call overriding the previous user.

The issue affects cases where REST Client is run several times within
the same thread.

With these changes we restore the original permission checker after the
REST Call to avoid side effects.

lfbesada committed:
LPD-1226 Apply the same fix on the TermsOfUseStrutsAction to prevent a NPE in case no element with ID "content"

vngoliveira committed:
LPD-45168 Match order in modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/resources/META-INF/sql/tables.sql

carolmariaabb committed:
LPD-45012 Show related Object Fields in the expression builder sidebar for Formula fields

carolmariaabb committed:
LPD-45012 Add integration test

carolmariaabb committed:
LPD-45012 Add query expressions for related object field columns when evaluating the DDMExpression for formula fields

carolmariaabb committed:
LPD-45012 Add integration test

pedro-oliveira446 committed:
LPD-44371 Build Service

kevenleone committed:
LPD-45313 Throw exception if status code is either 4xx or 5xx

ericyanLr committed:
LPD-45236 Add test to expose issue

Assert the loginDate and lastLoginDate is stored with the correct
date and timestamp when initially updated or when updated using the
BatchProcessor.

ericyanLr committed:
LPD-45236 Use setTimestamp to store both the date and respective time

Using setDate can properly store the date, but not the respective
time. It instead normalizes the time to 00:00:00.

ling-alan-huang committed:
LPD-44271 Auto SF

ling-alan-huang committed:
LPD-44271 Auto SF

ling-alan-huang committed:
LPD-44271 Auto SF

oliv-yu committed:
LPD-45174 portal-search-web: Close modal before saving blueprint/element

Commit:
LPD-44208 Jakarta cache

tinatian committed:
LPD-45236 Simplify test logic

michaelhashimoto committed:
LRCI-5161 Remove mirrors hostname references

michaelhashimoto committed:
LRCI-5161 Add method to set patch shas for hotfix tests

ling-alan-huang committed:
LPD-43790 Remove duplicate descriptions

ling-alan-huang committed:
LPD-43790 gw buildLang

ling-alan-huang committed:
LPD-43790 Change language keys

ling-alan-huang committed:
LPD-43790 Auto SF

ling-alan-huang committed:
LPD-43790 Remove duplicate descriptions

ling-alan-huang committed:
LPD-43790 gw buildLang

ling-alan-huang committed:
LPD-43790 Change language keys

ling-alan-huang committed:
LPD-43790 Auto SF

ling-alan-huang committed:
LPD-43477 SF

ling-alan-huang committed:
LPD-43477 SF

ling-alan-huang committed:
LPD-43477 SF

petershin committed:
LPD-44208 Output directory

petershin committed:
LPD-44208 Clean cache

petershin committed:
LPD-44208 Source formatting

brianchandotcom committed:
LPD-34518 prep next

brianchandotcom committed:
LPD-34518 prep next

brianchandotcom committed:
LPD-34518 Just master

kenjiheigel committed:
LRCI-5067 Ignore particular email error

brianchandotcom committed:
LRCI-5161 prep next

brianchandotcom committed:
LPD-44208 prep next

brianchandotcom committed:
LPD-45012 Rename

brianchandotcom committed:
LPD-44524 Do we test for custom?

brianchandotcom committed:
LPD-44987 Wordsmith

brianchandotcom committed:
LPD-44987 Regen

brianchandotcom committed:
LPD-45168 Order matters, so add a line break

brianchandotcom committed:
LPD-45168 Not needed?

brianchandotcom committed:
LPD-1223 SF

brianchandotcom committed:
LPD-44779 SF

brianchandotcom committed:
LPD-45266 Sort it

brianchandotcom committed:
LPD-42985 Not needed

brianchandotcom committed:
LPD-42985 Regen

brianchandotcom committed:
LPD-42985 SF

brianchandotcom committed:
LPD-42985 SF

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

sandrodw3 commented:
ci:test:playwright

liferay-continuous-integration commented:
<html><h3>:x: ci:test:playwright - 0 out of 1 jobs passed in 9 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/61b58512ab7f586f7ca6824d9a013acf90aafc3f">61b58512ab7f586f7ca6824d9a013acf90aafc3f</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/83a242f48ab4b2086965005a590a95e42e9c0a13">83a242f48ab4b2086965005a590a95e42e9c0a13</a><br/>Jenkins Build URL: <a href="https://testray.liferay.com/#/project/35392/routines/590307/build/103654631">EE Development Acceptance (master) - 1117 - 2025-01-04[12:36:18]</a></p><details><summary><strong>ci:test:playwright - 0 out of 1 jobs PASSED</strong></summary><div><h4>1 Failed Jobs:</h4><ul><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest(master)/13543/"><strike>test-portal-acceptance-pullrequest(master)</strike></a></li></ul></div></details><h5>For more details click <a href="https://test-1-27.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/13543/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><h5><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest(master)/13543/">test-portal-acceptance-pullrequest(master)</a></h5><div><h6>Job Results:</h6><p>0 Jobs Passed.<br/>1 Job Failed.</p></div><pre>Started by remote host 172.16.168.126
Running as SYSTEM
[EnvInject] - Loading node environment variables.
Building remotely on test-1-27-23 (slave-1) in workspace /opt/dev/projects/github/liferay-jenkins-ee
[liferay-jenkins-ee] \$ /bin/bash -e -x /tmp/jenkins9516669558677484227.sh
+ echo 'Initialize Jenkins slave node.'
Initialize Jenkins slave node.
+ cd /opt/dev/projects/github/liferay-jenkins-ee
+ curl -o ./_common.sh http://mirrors-no-cache.lax.liferay.com/github.com/liferay/liferay-jenkins-ee/_common.sh
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 19380  100 19380    0     0  2072k      0 --:--:-- --:--:-- --:--:-- 2102k
+ source ./_common.sh
++ MAX_THREADS=1
++ [[ test-1-27-23.lax.liferay.com =~ cloud-.* ]]
++ [[ test-1-27-23 slave-1 =~ .*win.* ]]
++ ROOT_DIR=
++ GIT_REPOSITORIES_ROOT_DIR=/opt/dev/projects/github
++ JENKINS_REPO_PATH=/opt/dev/projects/github/liferay-jenkins-ee
+++ cat /opt/dev/projects/github/liferay-jenkins-ee/git-directories.json
++ GIT_DIRECTORIES_JSON_CONTENT='[
  \{
    "branch": "master",
    "name": "alloy-ui",
    "repository": "alloy-ui",
    "type": "mounted"
  \},
  \{
    "branch": "7.0.x",
    "name": "com-liferay-osb-asah-private",
    "repository": "com-liferay-osb-asah-private",
    "type": "mounted"
  \},
  \{
    "branch": "master",
    "name": "liferay-binaries-cache-2017",
    "repository": "liferay-binaries-cache-2017",
    "type": "shared-mounted"
  \},
  \{
    "branch": "master",
    "name": "liferay-binaries-cache-2020",
    "repository": "liferay-binaries-cache-2020",
    "type": "shared-mounted"
  \},
  \{
    "branch": "master",
    "name": "liferay-blade-samples",
    "repository": "liferay-blade-samples",
    "type": "mounted"
  \},
  \{
    "branch": "master",
    "name": "liferay-docker",
    "repository": "liferay-docker",
    "type": "mounted"
  \},
  \{
    "branch": "master",
    "name": "liferay-faces-alloy",
    "repository": "liferay-faces-alloy",
    "type": "mounted"
  \},
  \{
    "branch": "master",
    "name": "liferay-faces-bridge-impl",
    "repository": "liferay-faces-bridge-impl",
    "type": "mounted"
  \},
  \{
    "branch": "master",
    "name": "liferay-faces-portal",
    "repository": "liferay-faces-portal",
    "type": "mounted"
  \},
  \{
    "branch": "master",
    "name": "liferay-faces-showcase",
    "repository": "liferay-faces-showcase",
    "type": "mounted"
  \},
  \{
    "branch": "master",</pre></div></li></ol><h4>For upstream results, click <a href="https://test-1-40.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">here</a>.</h4></details></html>
